internal/globalratelimit/runner/runner.go

```diff
@@ -36,62 +57,123 @@ func New(cfg *Config) *Runner {
 // Start starts the infrastructure runner
 func (r *Runner) Start(ctx context.Context) error {
 	r.Logger = r.Logger.WithValues("runner", r.Name())
+
+	// Set up the gRPC server for RateLimit xDS Config.
+	r.grpc = grpc.NewServer()
```

Comment:
do we need to enable a TLS connection with the xDS confi server?

Reply:
yes, can be done as a follow up

Reply:
raised https://github.com/envoyproxy/gateway/issues/1454 to track it